### ISSUE TITLE
Could not load 'plugins/MyPet-3.0-SNAPSHOT.jar'

### ISSUE THREAD

**Issue by PinoyLifeCraft** [xXKeyleXx/MyPet#1216]
I can't do `/petadmin ticket` since the plugin do not load to my server.

**Bukkit/Spigot/Bungee Build/Version:** Spigot

**MyPet Build/Version:3.0 SNAPSHOT (February 11, 2019)**

**Do you use the premium version? (this wont change anything for you)** No

**Error:Could not load 'plugins/MyPet-3.0-SNAPSHOT.jar' in folder 'plugins'
11.02 08:03:55 [Server] INFO org.bukkit.plugin.InvalidDescriptionException: Invalid plugin.yml
11.02 08:03:55 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader.getPluginDescription(JavaPluginLoader.java:163) ~[custom.jar:git-Spigot-f56e2e7-7fc66b2]
11.02 08:03:55 [Server] INFO at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:133) [custom.jar:git-Spigot-f56e2e7-7fc66b2]
11.02 08:03:55 [Server] INFO at org.bukkit.craftbukkit.v1_13_R2.CraftServer.loadPlugins(CraftServer.java:323) [custom.jar:git-Spigot-f56e2e7-7fc66b2]
11.02 08:03:55 [Server] INFO at net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:213) [custom.jar:git-Spigot-f56e2e7-7fc66b2]
11.02 08:03:55 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:698) [custom.jar:git-Spigot-f56e2e7-7fc66b2]
11.02 08:03:55 [Server] INFO at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
11.02 08:03:55 [Server] INFO Caused by: java.util.zip.ZipException: error in opening zip file
11.02 08:03:55 [Server] INFO at java.util.zip.ZipFile.open(Native Method) ~[?:1.8.0_101]
11.02 08:03:55 [Server] INFO at java.util.zip.ZipFile.<init>(ZipFile.java:219) ~[?:1.8.0_101]
11.02 08:03:55 [Server] INFO at java.util.zip.ZipFile.<init>(ZipFile.java:149) ~[?:1.8.0_101]
11.02 08:03:55 [Server] INFO at java.util.jar.JarFile.<init>(JarFile.java:166) ~[?:1.8.0_101]
11.02 08:03:55 [Server] INFO at java.util.jar.JarFile.<init>(JarFile.java:130) ~[?:1.8.0_101]
11.02 08:03:55 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader.getPluginDescription(JavaPluginLoader.java:151) ~[custom.jar:git-Spigot-f56e2e7-7fc66b2]
11.02 08:03:55 [Server] INFO ... 5 more**

**Comment by xXKeyleXx** [comment]
Could you upload the plugin jar here please: https://nextcloud.keyle.de/s/k7fRw63pnCwm6z7

**Comment by PinoyLifeCraft** [comment]
Upload done at https://nextcloud.keyle.de/s/k7fRw63pnCwm6z7

**Comment by PinoyLifeCraft** [comment]
Ok I figured it out. Its Filezilla's fault the file I uploaded was corrupted.

How I discovered?
I reupload the MyPet plugin then boom it works.
How will I support my discovery?
At the time I reuploaded the MyPet plugin I am also uploaded a new plugin and .yml file. I ran the server then the MyPet plugin worked then the new plugin I uploaded was not, same error ~ the "could not load plugins/<pluginname>.jar" and also the .yml uploaded was broken :joy: 

I now closed this issue - Issue Solved :+1:
